frontend/src/static/js/v_summary.js

```diff
@@ -525,19 +525,10 @@ window.vSummary = {
       this.tmpFilterUntilDate = this.maxDate;
     },
 
-    updateDateRange() {
-      if (drags.length > 0) {
-        const since = new Date(this.filterSinceDate).getTime();
-        const until = new Date(this.filterUntilDate).getTime();
-        const range = until - since;
-
-        const getStr = (time) => getDateStr(new Date(time));
-        this.tmpFilterSinceDate = getStr(since + range * drags[0] / 100);
-        this.tmpFilterUntilDate = getStr(since + range * drags[1] / 100);
-
-        drags = [];
-        deactivateAllOverlays();
-      }
+    updateDateRange(zoomSince, zoomUntil) {
+      this.tmpFilterSinceDate = zoomSince;
+      this.tmpFilterUntilDate = zoomUntil;
```

Comment:
Not sure if it is better to rename `zoomSince` to `since` and `zoomUntil` to `until` for generality